Tests/varLib/data/test_results/TestSparseCFF2VF.ttx

```diff
@@ -290,7 +290,7 @@
         </CharString>
         <CharString name="cid06449" fdSelectIndex="1">
           2 vsindex
-          -60 30 203 30 -9 9 67 7 -7 14 -14 30 -20 20 80 30 59 30 121 30 18 93 -30 30 -30 108 -23 0 -26 67 2 76 -98 -2 -111 42 0 47 -13 0 -14 13 0 14 -33 0 -37 11 0 13 -11 0 -13 8 0 9 -7 0 -8 53 0 60 -32 0 -36 32 0 36 -52 0 -59 57 1 65 -33 0 -38 53 0 60 -83 -1 -93 54 0 60 -6 -19 -24 33 19 55 -76 -1 -86 76 1 86 -76 -1 -86 59 1 67 26 blend
+          -60 30 203 30 -9 9 67 7 -7 14 -14 30 -20 20 80 30 59 30 121 30 18 93 -30 30 -30 108 -23 0 -26 67 2 76 -98 -2 -111 42 0 47 -13 0 -14 13 0 14 -33 0 -37 11 0 13 -11 0 -13 8 0 9 -7 -1 -8 53 0 60 -32 0 -36 32 0 36 -52 0 -59 57 1 65 -33 0 -38 53 0 60 -83 -1 -93 54 0 60 -6 -19 -24 33 19 55 -76 -1 -86 76 1 86 -76 -1 -86 59 1 67 26 blend
```

Comment:
not immediately obvious to me if this diff was intentional in the expected test result

Reply:
Right. This changed one 0 delta to -1. I assumed it's a result of the error accumulation changes I've made.

That said, there's a tweak I want to make to the rounding still; I'll comment below.

Reply:
Good news. Rounding deltas with round3 instead of otRound indeed reverted this change.